classes/install/Installer.php

```diff
@@ -949,8 +951,11 @@ public function checkPhpVersion()
      */
     public function updateRorRegistryDataset(): bool
     {
+        PKPApplication::upgrade();
```

Comment:
Had to add this as below class calls internally `PKPApplication::getHttpClient` which try to retrieve the version information but we register the version details at the very end of installation phase and without this get `null` which raise exception .